docs/appendix.rst

```diff
@@ -18,112 +18,77 @@ This documentation uses a few special terms to refer to Python types:
+      A CIM instance path in a format that is suitable as a command line
+      argument or option value of pywbemcli. Several commands of the instance
+      command group use INSTANCENAME as a command line argument.
+      The only supported format for INSTANCENAME is a :term:`WBEM URI`.
+      See also :ref:`Interactively selecting INSTANCENAME`.
 
-      When the ``interactive`` option is not set, the INSTANCENAME is a string
-      representation of a CIMInstanceName must be formatted as defined by
-      :term:`WBEM-URI`
+      Examples::
 
-      Otherwise the INSTANCENAME should be a classname in which case pywbemcli
-      will get the instance names from the WBEM server and present a
-      selection list for the user to select an instance
-      name :ref:`Displaying CIM instances or CIM instance names`
+        CIM_RegisteredProfile.InstanceID="acme:1"
+        CIM_System.CreationClassName="ACME_System",Name="MySystem"
 
    connection id
-      a string that uniquely identifies each :class:`pywbem.WBEMConnection`
-      object created. The connection id is immutable and is accessible from
-      :attr:`pywbem.WBEMConnection.conn_id`. It is included in of each log
-      record created for pywbem log output and may be used to correlate pywbem
-      log records for a single connection.
+      A string that uniquely identifies each :class:`pywbem.WBEMConnection`
+      object created. The connection id is immutable and is accessible as
+      :attr:`pywbem.WBEMConnection.conn_id`. It is included in each log
+      record and may be used to relate pywbem log records to connections.
 
    DeprecationWarning
-      a standard Python warning that indicates a deprecated functionality.
+      A standard Python warning that indicates a deprecated functionality.
       See section :ref:`Deprecation and compatibility policy` and the standard
       Python module :mod:`py:warnings` for details.
 
    CIM namespace
       A CIM namespace (defined in :term:`DSP0004`) provides a scope of
-      uniqueness  for cim objects; specifically, the names of class objects and
+      uniqueness for CIM objects; specifically, the names of class objects and
       of qualifier type objects shall be unique in a namespace. The compound
-      key of non- embedded instance objects shall also be unique across all
+      key of non-embedded instance objects shall also be unique across all
       non-embedded instances of the class (not including subclasses) within the
       namespace.
 
    connections file
-      A file maintained by pywbemcli and managed by the pywbemcli
-      ``connections`` command group.  The file name is ``pywbemcli_connections.json``
-      and the file must be in the directory in which pywbemcli is executed.
+      A file maintained by pywbemcli and managed by the
+      :ref:`Connection command group`.
+      The connections file name is ``pywbemcli_connections.json`` and is created
```

Comment:
I did not check for file name consistency. Sorry for that.

We say it contains "persistent WBEM connections", but that is actually quite misleading as a term, because a connection only exists once being connected. And there is nothing about this notion of a connection that we are persisting. Instead, we are persisting the information to connect to a WBEM server.

So maybe "persistent WBEM connection info" for the term and "pywbemcli_connection_info.json" for the file?

Reply:
I agree and will make this change.

1. use pywbemcli-connection_definitions.json

2. WBEM connection definitions and remove the work persistent.

Reply:
In a separate issue, we want to change the file format from json to yaml.

Reply:
DONE as proposed by Karl.